test/vandermonde.jl

```diff
@@ -1,46 +1,68 @@
+# vandermonde.jl tests
+
+using SpecialMatrices: Vandermonde
+import SpecialMatrices # dvand!, pvand!
+using Test: @test, @testset, @test_throws, @inferred
+
```

Comment:
These are not needed since `runtests.jl` loads SpecialMatrices and Test, and everything is exported.
```suggestion
```

Reply:
Yes I know but I like to be able to just `include("vandermonde.jl")` when developing and have it be a self-contained test.  And I feel that Julia code is more readable when you see the dependencies given explicitly at the top of each file.  I agree it is not needed, but no real drawback, right?

Reply:
No, it's not completely uncommon. It's mainly surprising that all imported macros etc. are listed explicitly instead of
```suggestion
using SpecialMatrices
using Test
```
as in runtests.jl. I assumed the intention was to not pollute the namespace which does not work since runtests.jl loads SpecialMatrices and Test.

Reply:
Yes I know - I'll work my way up to runtests.jl eventually.
After literally decades of using Matlab with no namespace control, perhaps I am overly excited about the opportunity to be very explicit 😃